### ISSUE TITLE
A snapshot of the scanned code

### ISSUE THREAD

**Issue by Wilker-a** [markusfisch/BinaryEye#150]
a suggestion for a feature where the program shows a snapshot of the relevant image of the scanned code, for error checking purposes, specially useful when barcodes are read improperly, for example. :3

**Comment by markusfisch** [comment]
Yes, this is possible, and was already requested so I will definitely add that 😄

**Comment by oddstr13** [comment]
I'd love an option for a copy of the actual image used to decode be sent as part of the json post payload.
This to make it possible to do additional processing on data that may be located near the barcode, such as running OCR for ISBN next to a non-ISBN barcode.
(A button to send just whatever is in the camera view now would also be nice, but probably out of scope for this app)
